assets/js/services/helpers.js

```diff
@@ -18,10 +19,14 @@ export function formatDate(date) {
 export function formatSimpleISODate(date) {
   if (!date) return "";
   let newDate = moment(date).format().substring(0, 16);
-
   return newDate;
 }
 
+export function isUrlValid(url) {
+  const res = url.match(URL_PATTERN_MATCH);
+  return res ? true : false;
```

Comment:
If you wanted, we could combo these lines into:

`return url.match(URL_PATTERN_MATCH);`

Reply:
Also, can we add a quick unit test for this function in `assets/js/services/helpers.test.js`

Reply:
`url.match()` returns null when empty. I updated the code to a ternary instead of using a `const`. Added a test case under helpers.test.js for this function and possible outcomes.